src/Common/src/TypeSystem/Common/TargetDetails.cs

```diff
@@ -33,6 +33,22 @@ public enum TargetOS
     }
 
     /// <summary>
+    /// Specifies 
+    /// </summary>
+    public enum TargetApplicationModel
+    {
+        Unknown,
+        /// <summary>
+        /// Cross-platform console model
+        /// </summary>
+        CoreRT,
+        /// <summary>
+        /// Windows-specific UWP model
+        /// </summary>
+        NetNative
```

Comment:
ProjectN is not the official marketing name, but we use it as codename in many places; and have it documented here: https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/glossary.md